syslog/bin/import_logs.py

```diff
@@ -1115,6 +1113,7 @@ def _get_token_auth(self):
 
             filename = stdout
             credentials = open(filename, 'r').readline()
+            filename.close()
```

Comment:
Should this be closed here?

Reply:
Yep, though the risk is super low since this code isn't web-accessible